src/foam/nanos/u2/navigation/UserInfoNavigationView.js

```diff
@@ -71,7 +71,7 @@ foam.CLASS({
             .add(this.slot((agent, user) => {
               return this.E().addClass('name-container')
                   .start('span').addClass(this.myClass('userName'))
-                    .add( agent ? user.businessName() : user.label())
+                    .add( agent.firstName ? user.businessName : user.label())
```

Comment:
User.businessName is not required so perhaps the test and output should be: 
```suggestion
                    .add( agent.businessName ? agent.businessName : user.label())
```

Reply:
I don't think agent.businessName makes much sense to check here since agents could be apart of multiple users also the purpose of this condition is if agent exists/has a displayable value(firstName), we use user(Business) business name, instead of first and last name. We account for both states. One displaying a business name the other a users first name and last name.

Then the slot on ln 77 accounts for the agent and displays the appropriate label depending on it state.

Here's the design I'm referencing https://app.zeplin.io/project/5cf963f36af3e11bb4bedf5f/screen/5cf9674986ef376f6070fe8e

Reply:
Deferring to Troy below.